packages/cli/src/index.ts

```diff
@@ -143,8 +143,10 @@ async function main() {
       if (!selectedComponents?.length) {
         selectedComponents = await promptForComponents(availableComponents)
       }
-
-      const dir = await promptForDestinationDir()
+            
+      const dir = await promptForDestinationDir(
```

Comment:
This looks good. Note: I'm also doing some work to handle this. I'll commit this for now.